apps/connect-v1/vite.token-bridge.config.ts

```diff
@@ -30,7 +32,7 @@ export default defineConfig({
             injectTo: "head-prepend",
             tag: "meta",
             attrs: {
-              name: "Portal Bridge",
+              name: "Cosmos Token Bridge",
```

Comment:
from where this name comes from?

Reply:
From apps/redirects/src/configs/cosmos.env

Reply:
I would like to keep the title, but I know that if I follow the example of the others it would be something more like: Portal Cosmos Bridge.